frontend/src/components/crd/ResourceDetails.tsx

```diff
@@ -0,0 +1,137 @@
+import { JSONPath } from 'jsonpath-plus';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import DetailsViewPluginRenderer from '../../helpers/renderHelpers';
+import { ApiError } from '../../lib/k8s/apiProxy';
+import CRD, { KubeCRD, makeCustomResourceClass } from '../../lib/k8s/crd';
+import { localeDate } from '../../lib/util';
+import { HoverInfoLabel, NameValueTableRow, SectionBox } from '../common';
+import Empty from '../common/EmptyContent';
+import Loader from '../common/Loader';
+import { ConditionsTable, MainInfoSection, PageGrid } from '../common/Resource';
+
+/* istanbul ignore next */
+export default function CustomResourceDetails() {
+  const {
+    crd: crdName,
+    namespace: ns,
+    crName,
+  } = useParams<{
+    crd: string;
+    crName: string;
+    namespace: string;
+  }>();
+  const [crd, setCRD] = React.useState<CRD | null>(null);
+  const [error, setError] = React.useState<ApiError | null>(null);
+
+  const { t } = useTranslation('glossary');
+
+  const namespace = ns === '-' ? undefined : ns;
+
+  CRD.useApiGet(setCRD, crdName, undefined, setError);
+
+  return !crd ? (
+    !!error ? (
+      <Empty color="error">
+        {t(`crd|Error getting custom resource definition ${crdName}: ${error.message}`)}
+      </Empty>
+    ) : (
+      <Loader title={t('crd|Loading custom resource details')} />
+    )
+  ) : (
+    <CustomResourceDetailsRenderer crd={crd} crName={crName} namespace={namespace} />
+  );
+}
+
+type AdditionalPrinterColumns = KubeCRD['spec']['versions'][0]['additionalPrinterColumns'];
+
```

Comment:
same as earlier this won't be required if we add a story for CRD detail view

Reply:
Why wouldn't this be required if we had a story?